tasks/shell_crmsh/crm-cib-resource-clone.yml

```diff
@@ -20,24 +31,31 @@
 # Delete resource id in Shadow CIB to avoid errors during cibadmin patch.
 - name: Delete present resouce clone {{ __ha_cluster_resource_id }}
   ansible.builtin.command:
-    cmd: |
-      crm -c {{ __ha_cluster_crm_shadow }}
+    cmd: >-
+      crm --force -c {{ __ha_cluster_crm_shadow }}
       configure delete {{ __ha_cluster_resource_id }}
   when: __ha_cluster_resource_status.rc == 0
   check_mode: false
   changed_when: not ansible_check_mode
 
+# Clone is default resource type, unless attribute ms: true is specified
+# Pacemaker 2.0 deprecated use of ms, but it is still valid and supported
+# solution for SAP Hana clusters on SUSE using SAPHanaSR.
 - name: Configure resource clone {{ __ha_cluster_resource_id }}
   ansible.builtin.command:
     cmd: |
```

Comment:
```suggestion
    cmd: >-
```

Reply:
@richm I have changed pipe to >- where it was possible and I am keeping it for multi-line commands only. Can we close pipe discussion?